functions/Set-DbaTempDbConfiguration.ps1

```diff
@@ -181,15 +173,15 @@ Returns PSObject representing tempdb configuration.
 			$Filepath = $server.Databases['tempdb'].ExecuteWithResults('SELECT physical_name as FileName FROM sys.database_files WHERE file_id = 2').Tables[0].Rows[0].FileName
 			$LogPath = Split-Path $Filepath
 		}
-		Write-Message -Message "Using log path: $LogPath" -Level Verbose
+		Write-Message -Message "Using log path: $LogPath." -Level Verbose
 		
 		# Check if the file growth needs to be disabled
 		if ($DisableGrowth) {
 			$DataFileGrowthMB = 0
 			$LogFileGrowthMB = 0
 		}
 		
-		$LogSizeMBActual = if (-not $LogFileSizeMB) { $([Math]::Floor($DataFileSizeMB/4)) }
+		$LogSizeMBActual = if (-not $LogFileSizeMB) { $([Math]::Floor($DataFileSizeMB / 4)) }
```

Comment:
Was this formatting by hand or do you have the format set in your editor to do this?

Reply:
That must have been the auto-format in VSCode.